generators/app/templates/express-api/Dockerfile

```diff
@@ -1,11 +1,12 @@
-FROM alpine:3.3
```

Comment:
In terms of knowing what all is inside the image, knowing version of node, or both? I wonder if there is any size difference. Bet node one is bigger